sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ThroughputControlGroupConfigBuilder.java

```diff
@@ -157,6 +158,21 @@ public ThroughputControlGroupConfigBuilder defaultControlGroup(boolean aDefault)
         return this;
     }
 
+    /**
+     * Set the throughput bucket of the group.
+     * <p>
+     * For more information about throughput bucket please visit
+     * <a href="https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/throughput-buckets?tabs=dotnet">Throughput buckets in Azure Cosmos DB</a>
+     *
+     * @param throughputBucket the throughput bucket id.
+     * @return The {@link ThroughputControlGroupConfigBuilder}.
+     */
+    public ThroughputControlGroupConfigBuilder throughputBucket(int throughputBucket) {
+        checkArgument(throughputBucket > 0, "Throughput Bucket should be greater than 0");
```

Comment:
Please double-check - I think 0 is viable (it is the throughput bucket the portal also uses - but there is no reason ahy it can't be used by apps as well. If Portal uses 1 - maybe 0 is really invalid.

Reply:
portal is using 1 -> but I think I see your point, this checking should be guarded by the backend, will change to just check >= 0

Reply:
updated the check to >= 0